CMakeLists.txt

```diff
@@ -135,6 +135,8 @@ include(iree_check_test)
 
 string(JOIN " " CMAKE_CXX_FLAGS ${IREE_DEFAULT_COPTS})
 
+# FastBuild requires -DCMAKE_C_COMPILER=clang and -DCMAKE_CXX_COMPILER=clang++
+# (or the CC/CXX environment variables) to be specified.
```

Comment:
I don't think this should be special to fastbuild and it's really a general cmake thing that you have to set the compiler somehow. Or are you saying that our fastbuild config specifically requires clang (I guess `gmlt` is only clang?). We only support clang in general and we should probably change this to just use `RelWithDebInfo` probably.

Basically, I would drop this comment or just say "FastBuild only works with clang"

Reply:
yeah, we need to remove "fastbuild" - RelWithDebInfo is the canonical name